plugins/USBPrinting/FirmwareUpdateWindow.qml

```diff
@@ -32,20 +32,51 @@ UM.Dialog
             }
 
             text: {
-                if (manager.firmwareUpdateCompleteStatus)
+                if (manager.errorCode == 0)
                 {
-                    //: Firmware update status label
-                    return catalog.i18nc("@label","Firmware update completed.")
-                }
-                else if (manager.progress == 0)
-                {
-                    //: Firmware update status label
-                    return catalog.i18nc("@label","Starting firmware update, this may take a while.")
-                }
-                else
+                    if (manager.firmwareUpdateCompleteStatus)
+                	{
+	                    //: Firmware update status label
+    	                return catalog.i18nc("@label","Firmware update completed.")
```

Comment:
Used the regular text editor of Eclipse for QML files. Possibly it just enabled tabs again. However, I'll fix that.